test/tools/isoltest.cpp

```diff
@@ -29,6 +29,10 @@
 #include <fstream>
 #include <queue>
 
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
Isn't ``_WIN32`` set on 64-bit windows as well?

Reply:
@ekpyron that may be true. Maybe a Windows-(pro){2}gramma can confirm that, otherwise I'll digg into it as soon as I'm back home and free. :)

Reply:
If you find that it isn't, then we need to add the second disjunct at some other places as well... but I'm pretty sure that ``_WIN32`` will be set everywhere (but I'm no expert for Windows code).

Reply:
So I checked, _WIN32 is defined regardless of the architecture, literally meaning "You're using Windows API", however _WIN64 is set in addition when you're compiling unter 64-bit mode.

Reply:
Yep, that's what I thought.

Reply:
In other files we check for `_WIN32` and the include `windows.h` (lowercase)